policyengine_us/variables/gov/states/ct/tax/income/add_back/ct_income_tax_higher_tax_recapture.py

```diff
@@ -0,0 +1,24 @@
+from policyengine_us.model_api import *
+from numpy import ceil
+
+
+class ct_income_tax_higher_tax_recapture(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Connecticut income tax higher tax recapture"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.CT
+
+    def formula(tax_unit, period, parameters):
+        income = tax_unit("ct_agi", period)
+        p = parameters(period).gov.states.ct.tax.income.main.recapture.higher
+        filing_status = tax_unit("filing_status", period)
+        reduction_start = p.start[filing_status]
+        max_amount = p.max_amount[filing_status]
+        increment = p.increment[filing_status]
+        reduction_amount = p.amount[filing_status]
+        income_start = max_(income - reduction_start, 0)
+        income_brackets = ceil(income_start / increment)
```

Comment:
Check the logic and ensure you're testing appropriately
```suggestion
        increments = ceil(excess / increment)
```